### DISCUSSION TITLE
How to remove unwanted content-type of 'text/plain;charset=UTF-8' from $fetch 'post' ?

### DISCUSSION THREAD

**Discussion by ian4uia** [nuxt/nuxt#17429] [Questions]
Hi, I need to create a post request with the following headers:
```
prefer: params=single-object
content-type: application/json
```
I am using this code:
```
  const data = await $fetch('http://http-logger:3001/rpc/test_proc', {
    method: 'post',
    headers: {
      Prefer: 'params=single-object',
      'Content-Type': 'application/json'
    },
    body: {some: 'json'}
  })
```
This produces the following http request:
```
POST /rpc/test_proc HTTP/1.1
host: http-logger:3001
connection: keep-alive
prefer: params=single-object
content-type: application/json, text/plain;charset=UTF-8
accept: application/json
accept-language: *
sec-fetch-mode: cors
user-agent: undici
accept-encoding: gzip, deflate
content-length: 15


{"some":"json"}

```
However `text/plain;charset=UTF-8` is getting appended to the content-type.  How can this be removed?  My post request is being rejected because of this unwanted setting.
Many thanks
Ian

**Comment by Flanker72** [comment]
Seems you have any non-nuxt and non-ohmyfetch issue. Possible something locally modifies your headers.
Calling exactly the same request doesn't add anything in addition to passed headers.

<img width="900" alt="image" src="https://user-images.githubusercontent.com/619699/174140734-97d2bcfe-a8f2-4ae6-932d-6b0b8791c15f.png">
